src/Illuminate/Auth/Passwords/TokenRepositoryInterface.php

```diff
@@ -23,6 +23,14 @@ public function create(CanResetPasswordContract $user);
      */
     public function exists(CanResetPasswordContract $user, $token);
 
+    /**
+     * Determine if a token record exists and was recently created.
+     *
+     * @param \Illuminate\Contracts\Auth\CanResetPassword $user
+     * @return bool
+     */
+    public function recentlyCreated(CanResetPasswordContract $user);
```

Comment:
By adding it to the interface, this should definitely go to 7.x. You should update the target branch to `master`.

If you want to merge it to 6.x then you need to check with `method_exists` and a second pull request can add this to the interface and remove the check.